CGPs/0023.md

```diff
@@ -10,43 +10,42 @@
 
 This governance proposal enables oracle addresses to make them able to report the value of CELO/EUR for the cEUR contract, deployed on [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md). It also proposed to send 100 CELO to each of the oracles to cover gas fees for the first couple of years and freezing StableTokenEUR (cEUR) and ExchangeEUR.
 
-Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (assuming no oracle reports, but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
+Following [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md), cEUR will be in an status where it would be imposible to mint or burn due empty buckets that are not possible to update (because Mento buckets are zero and there are no oracles enable to updated), but cEUR will be possible to move. Freezing is meant to prevent cEUR from moving before enabling the stability mechanism.
 
-cEUR in circulation with no stability mechanism in place can lead to a depledge. Although the risk of cEUR moving before its full activation it's very low, this proposal will add an extra ledger of security. The only to address holding cEUR after CR3 is deployed will by the community fund (Governance Proxy) and a Celo Foundation wallet.
+cEUR in circulation with no stability mechanism in place can lead to a depledge. Although the risk of cEUR moving before its full activation it's very low, this proposal will add an extra layer of security. The only two addresses holding cEUR after CR3 is deployed will by the [Community Fund](https://celocommunityfund.org/) (Governance Proxy) and a Celo Foundation wallet.
 
 Please note this proposal shall be submited after CR3 is proposed, because it needs an address for StableTokenEUR, but it can be executed before or after.
-
+ 
 [CGP #26](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md) was merged into this proposal after Baklava deployment.
 
-## Proposed Changes
 
-Fill out the following template for each transaction in the proposal
+## Proposed Changes
 
-1. The first X transactions enable a specific address to report to StableTokenEUR
+1. The first ten transactions enable a specific address to report to StableTokenEUR
```

Comment:
nit: make it clear that this is enabling multiple addresses to report to SortedOracles for the StableTokenEUR rate